ex/high_throughput_inference/standalone_workermanager.py

```diff
@@ -24,26 +24,118 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# isort: off
+
 import dragon
+
+# pylint disable=import-error
+import dragon.globalservices.pool as dragon_gs_pool
+import dragon.infrastructure.policy as dragon_policy
+import dragon.infrastructure.process_desc as dragon_process_desc
+import dragon.native.process as dragon_process
 from dragon import fli
 from dragon.channels import Channel
 from dragon.data.ddict.ddict import DDict
-from dragon.utils import b64decode, b64encode
 from dragon.globalservices.api_setup import connect_to_infrastructure
+from dragon.managed_memory import MemoryPool
+from dragon.utils import b64decode, b64encode
+# pylint enable=import-error
+
+# isort: off
 # isort: on
+
 import argparse
 import base64
-import cloudpickle
-import pickle
+import multiprocessing as mp
 import os
+import pickle
+import socket
+import sys
+import time
+import typing as t
+
+import cloudpickle
 
+from smartsim._core.entrypoints.service import Service
+from smartsim._core.mli.comm.channel.channel import CommChannelBase
 from smartsim._core.mli.comm.channel.dragonchannel import DragonCommChannel
-from smartsim._core.mli.infrastructure.storage.dragonfeaturestore import DragonFeatureStore
 from smartsim._core.mli.comm.channel.dragonfli import DragonFLIChannel
-from smartsim._core.mli.infrastructure.worker.torch_worker import TorchWorker
+from smartsim._core.mli.infrastructure.control.requestdispatcher import (
+    RequestDispatcher,
+)
 from smartsim._core.mli.infrastructure.control.workermanager import WorkerManager
 from smartsim._core.mli.infrastructure.environmentloader import EnvironmentConfigLoader
+from smartsim._core.mli.infrastructure.storage.dragonfeaturestore import (
+    DragonFeatureStore,
+)
+from smartsim._core.mli.infrastructure.worker.worker import MachineLearningWorkerBase
+
+mp.set_start_method("dragon")
+
+pid = os.getpid()
+affinity = os.sched_getaffinity(pid)
+print("Entry point:", socket.gethostname(), affinity)
+print("CPUS:", os.cpu_count())
+
+
+def create_request_dispatcher(
+    batch_size: int,
+    batch_timeout: float,
+    comm_channel_type: t.Type[CommChannelBase],
+    worker_type: t.Type[MachineLearningWorkerBase],
+    config_loader: EnvironmentConfigLoader,
+) -> RequestDispatcher:
+    mem_pool = MemoryPool.attach(dragon_gs_pool.create(2 * 1024**3).sdesc)
```

Comment:
1. memory pool size appears to be decoupled from the batch_size. 
- should it be related to ensure that a batch is not larger than the pool?

2. can `__init__` for `RequestDispatcher` handle creating a memory pool if one is not passed to avoid the need for `create_request_dispatcher`?